packages/core/src/components/popover/popoverSharedProps.ts

```diff
@@ -14,6 +14,13 @@ export { PopperModifiers };
 
 /** Props shared between `Popover` and `Tooltip`. */
 export interface IPopoverSharedProps extends IOverlayableProps, IProps {
+    /**
+     * Determines the boundary element used by Popper for its `flip` and
+     * `preventOverflow` modifiers.
+     * @default "viewport"
```

Comment:
`@default` is actually  "scrollParent" in the component (it's "viewport" in the example) (or is it  "window"?)